code/src/index.css

```diff
@@ -11,3 +11,155 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.app {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  word-wrap: break-word;
+  margin-top: 100px;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+}
+
+
+form {
+  display: flex;
+  flex-direction: column;
+  font-family: 'Roboto Mono', monospace;
+  background-color: #e8e9eb;
+  width: 70%;
+  padding: 30px; 
+  border: 1px solid black;
+  box-shadow: 10px 10px 0px 0px rgba(0,0,0,1);
+-webkit-box-shadow: 10px 10px 0px 0px rgba(0,0,0,1);
+-moz-box-shadow: 10px 10px 0px 0px rgba(0,0,0,1);
+}
+
+.form-conatiner {
+  display: flex;
+  flex-direction: column;
+}
+
+.form-conatiner label{
+  display: flex;
+  flex-direction: column;
+}
+
+.form-button {
+  width: 90%;
+  margin-top: 50px;
+  border: none;
+  border-radius: 25px;
+  color: #000;
+  background-color: #d9fcc2;
+  padding: 10px 15px;
+  font-size: 16px;
+  cursor: pointer; 
+}
+
+.heart-button {
+  border-radius: 50%;
+  width: 50px;
+  height: 50px;
+  margin-right: 5px;
+  border: none;
+  cursor: pointer; 
+}
+
+.heart-button-icon {
+  font-size: 18px;
+}
+
+.heart-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+
+.thoughts-length-left {
+  display: flex;
+  justify-content: flex-end;
+}
+
+
+/** Prevoius messages **/
+
+.list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 50px;
+}
+
+.previous-messages {
+  display: flex;
+  flex-direction: column;
+  font-family: 'Roboto Mono', monospace;
+  background-color: #fff;
+  width: 60%;
+  min-height: 80px;
+  margin-top: 50px;
+  padding: 20px 40px; 
+  border: 1px solid black;
+  box-shadow: 10px 10px 0px 0px rgba(0,0,0,1);
+-webkit-box-shadow: 10px 10px 0px 0px rgba(0,0,0,1);
+-moz-box-shadow: 10px 10px 0px 0px rgba(0,0,0,1);
+}
+
+.message-time {
+  display: flex;
+  justify-content: flex-end; 
+  padding-top: 20px;
+  font-size: 15px;
+  color: #b3b2b1;
+}
+
+
+
+@media screen and (min-width: 667px) and (max-width:1024px) { 
+
+  form {
+    width: 65%;
+  }
+  .previous-messages {
+    width: 65%;
+  }
+
+  .form-button {
+    width: 40%;
+    font-size: 14px;
+    padding: 10px 18px;
+     
+  }
+}
+
+
+
+/** MEDIA QUERIES DESKTOP **/
+
+@media screen and (min-width: 1024px) {
+
+  main {
+    width: 80%;
+  }
+  form {
+    width: 60%;
+  }
+
+  .previous-messages {
+    width: 60%;
+  }
+
+  .form-button {
+  width: 40%;
+  font-size: 16px;
+  padding: 15px 32px;
+  }
+ }
```

Comment:
Nicely done with responsiveness ✨ you can fix indentation on the css properties as well, one step to the right